inspirehep/modules/records/jsonschemas/records/experiments.json

```diff
@@ -14,13 +14,25 @@
             "type": "string"
         },
         "affiliation": {
-            "description": "Institution ICN",
-            "title": "Affiliation",
-            "type": "array"
-        },
-        "affiliation_record": {
-            "title": "Institution record URI",
-            "$ref": "json_reference.json"
+            "items": {
+                "properties": {
+                    "curated_relation": {
+                      "type": "boolean"
+                    },
+                    "name": {
+                        "title": "Name of the institution",
+                        "type": "string"
+                    },
+                    "record": {
+                        "title": "Institution's record URI",
+                        "$ref": "json_reference.json"
+                    }
+                },
+                "title": "Affiliation",
+                "type": "object"
+            },
+            "type": "array",
```

Comment:
Maybe it's just me, but I'd prefer to see the `type` key first, followed by `items`, finally `uniqueItems`. I dislike the `title`s that basically repeat the name of the key, and I'd use them only when the name of the key is correct but kind of obscure.